src/hyperlight_host/src/hypervisor/kvm.rs

```diff
@@ -55,6 +65,527 @@ pub(crate) fn is_hypervisor_present() -> bool {
     }
 }
 
+#[cfg(gdb)]
+mod debug {
```

Comment:
Is there any chance we could move this to it's own file? I understand kvm_guest_debug struct is kvm-specific, but otherwise the rest seems pretty generic to me, and to me it makes sense moving the gdb specific stuff out of the kvm.rs driver code. Hopefully it's possible :P

Reply:
Well, part of it is possible, the KvmDebug type specifically, however the rest of the implementation is for the `KVMDriver` so it cannot be moved out of the file the type is defined, right?